packages/core/integrations/integration-router.js

```diff
@@ -362,6 +370,59 @@ function setEntityRoutes(router, factory, getUserId) {
             }
         })
     );
+
+    router.route('/api/entities/:entityId').get(
+        catchAsyncError(async (req, res) => {
+            const params = checkRequiredParams(req.params, ['entityId']);
+            const module = await moduleFactory.getModuleInstanceFromEntityId(
+                params.entityId,
+                getUserId(req)
+            );
+
+            if (!module) {
+                throw Boom.notFound();
+            }
+            res.json(module.entity);
+        })
+    );
+
+    router.route('/api/entities/:entityId/options').post(
+        catchAsyncError(async (req, res) => {
+            const params = checkRequiredParams(req.params, [
+                'entityId',
+                getUserId(req)
+            ]);
+            const module = await moduleFactory.getModuleInstanceFromEntityId(
+                params.entityId,
+                getUserId(req)
+            );
+
+            if (!module) {
+                throw Boom.notFound();
+            }
+
+            res.json(await module.getEntityOptions());
```

Comment:
Small nit, but we're just doing res.json with the awaited function here, whereas above we're setting that to a const and res.json-ing the const.

Feels like style consistency is going to be key to be more strict on for Frigg core? At least, that's my instinct. Not a show stopper though. (Part of me likes defining the const and passing that along, I think largely because I get to name it and that lets me read and understand the intent if we're better with the names. Like here it would be const entityOptions... though we did a good job naming the method, so... 🤷 )